OpenSim/Actuators/Test/testMuscles.cpp

```diff
@@ -124,23 +124,23 @@ int main()
     try { testRigidTendonMuscle();
         cout << "RigidTendonMuscle Test passed" << endl; }
     catch (const Exception& e)
-        { e.print(cerr); failures.push_back("testRigidTendonMuscle"); }
+        { e.print(cout); failures.push_back("testRigidTendonMuscle"); }
```

Comment:
Is there a convention that we follow in all tests to use cout vs. cerr?

Reply:
I think tests should write to cout so the messages can be read directly in the console. I don't see the point of writing to cerr in test cases where the intention is to find (and see) errors.